addons/breathing/stringtable.xml

```diff
@@ -676,6 +677,7 @@
             <Italian>Quantità di sanguinamento interno da emopneumotorace</Italian>
             <Portuguese>Quantidade de hemorragia interna causada por hemopneumotórax</Portuguese>
             <French>Quantité de saignements internes de l'hémopneumothorax</French>
+            <German>Hämopneumothorax inneres Bluten menge </German>
```

Comment:
Suggestion:
Hämopneumothorax - Menge verursacht von inneren Blutungen

Reply:
I think as a option name it is fine as it is atm, I think the important thing to note here is that you don't have to structure it into a sentence due to that the option name should be short and compact and match the english one.

Reply:
If you say so, sure, but it still reads like a car accident and Menge needs to be capitalised.